README.md

```diff
@@ -37,7 +37,11 @@ install `aspire` safely in that environment.
 
 If you are unfamiliar with `conda`, the
 [Miniconda](https://docs.conda.io/en/latest/miniconda.html)
-distribution is recommended.
+distribution for `x86_64` is recommended.  For Apple silicon to use
+the `osx-arm` platform will currently require patching and building
+some dependencies from source.  The Intel `osx-64` install is still
```

Comment:
This sentence needs rewording: "For Apple silicon to use
the `osx-arm` platform, patching and building
some dependencies from source is currently required."

Reply:
updated